outputsheets/d20/fantasy/pdf/common_sheet/block_bio_condensed.xslt

```diff
@@ -50,6 +50,7 @@
 <!-->		<xsl:if test="string-length(translate(normalize-space(concat(description,bio)), ' ', '')) &gt; 0">	-->
 			<fo:page-sequence>
 				<xsl:attribute name="master-reference">Portrait</xsl:attribute>
+				<xsl:attribute name="font-family">NotoSans</xsl:attribute>
```

Comment:
I'm seeing this repeated, is there no way to place this into a global var and allow it to be set in one location? Otherwise, a lot of room for missing the font family.